pkg/handlers/login.go

```diff
@@ -385,6 +385,14 @@ func (h *Handler) OIDCLoginCallback(w http.ResponseWriter, r *http.Request) {
 	}
 	http.SetCookie(w, &tokenCookie)
 
+	signedTokenCookie, err := session.GetSessionCookie(responseToken, expire, redirectURL)
```

Comment:
this doesn't look right. the identity service token cookie is already set above, and that's the one that should be used.